core/model/upgradeseries.go

```diff
@@ -10,6 +10,18 @@ import (
 // UpgradeSeriesStatus is the current status of a series upgrade for units
 type UpgradeSeriesStatus string
 
+var UpgradeSeriesStatusOrder map[UpgradeSeriesStatus]int = map[UpgradeSeriesStatus]int{
```

Comment:
I don't think we need to export this.

Reply:
We export it for testing. I think it is harmless to export this for testing. Otherwise I have to figure out a way to test the ordering comparitor without knowing the order :-)